### ISSUE TITLE
Add z-index by default

### ISSUE THREAD

**Issue by everson** [aulizko/delicate-error-reporter#4]
When using in components that have z-index defined, this expanded error window is behind of some the components.

adding style z-index: 2000 to the root component made it appear. 

btw, Thanks for this plugin!

**Comment by everson** [comment]
oh, I see on master that it was been added. However, this is the style that gets applied to it:

``` css
.container_j2c_398286940_1442125373240_2 {
    overflow-y: auto;
    border: 10px solid #FF9966;
    color: #333;
    background: white;
    width: 100%;
    left: 0;
    bottom: 0;
    top: 0;
    padding: 10px;
    position: fixed;
    font-size: 16px;
    font-family: sans-serif;
    box-sizing: border-box;
}
```

I am using version 0.0.6

**Comment by aulizko** [comment]
Hey @khronnuz 
Just published 0.0.9 with some cleanup (your pull-request included).
It works on my local machine, hope that it'll work for you too. Go check it ;)

> btw, Thanks for this plugin!

You're welcome :)

**Comment by everson** [comment]
Awesome, I am already using 0.0.9 ;). thanks!
